file_types.go

```diff
@@ -0,0 +1,103 @@
+package rpmpack
+
+type fileType int32
+
+// https://refspecs.linuxbase.org/LSB_3.1.1/LSB-Core-generic/LSB-Core-generic/pkgformat.html#AEN27560
+// The RPMFile.Type tag value shall identify various characteristics of the file in the payload that it describes.
+// It shall be an INT32 value consisting of either the value GenericFile (0) or
+// the bitwise inclusive or of one or more of the following values. Some of these combinations may make no sense
+const (
+	// GenericFile is just a basic file in an RPM
+	GenericFile fileType = 0
+	// ConfigFile is a configuration file, and an existing file should be saved during a
+	// package upgrade operation and not removed during a package removal operation.
+	ConfigFile fileType = 1<<iota - 1
```

Comment:
What you were looking for is 
`1 << iota >> 1`
for example:
https://play.golang.org/p/3IU-0KLnX0X
This handles the zero case gracefully together with the rest.